2214/2/battle_expert.rb

```diff
@@ -0,0 +1,94 @@
+require 'commander'
+require 'terminal-table'
+require_relative 'bad_words_counter'
+require_relative 'words_in_round_counter'
+require_relative 'popular_words_counter'
+require_relative 'battler'
+require_relative 'battle'
+
+class BattleExpert
+  BATTLES_FOLDER = 'Battles'.freeze
+  def initialize
+    @battles = take_battles
```

Comment:
Let's not initialize instance variables in the initializer. I would remove this initialization and rename `#take_battles` to `#battles`. So we can use `#battles` method instead of `@battles` variable.

The same with `@battler_names`.